jwst/outlier_detection/_fileio.py

```diff
@@ -1,17 +1,11 @@
-import os
-
+import re
 import logging
+from jwst.datamodels import ImageModel
 log = logging.getLogger(__name__)
 log.setLevel(logging.DEBUG)
 
 
-def remove_file(fn):
-    if isinstance(fn, str) and os.path.isfile(fn):
-        os.remove(fn)
-        log.info(f"Removing file {fn}")
-
-
-def save_median(median_model, make_output_path, asn_id=None):
+def save_median(median_data, median_wcs, example_model, make_output_path=None):
```

Comment:
I think this is used in the other modes:
https://github.com/spacetelescope/jwst/blob/8091a35891e13feb7023e9317441632b3d3b5fb5/jwst/outlier_detection/coron.py#L59

Reply:
Let me know how this looks to you in the most recent version, where `asn_id` is no longer a parameter